ci/release/changelogs/next.md

```diff
@@ -1,5 +1,7 @@
 #### Features 🚀
 
+- Multi-byte characters (e.g. Chinese, Japanese, Korean, emojis) are supported. [#817](https://github.com/terrastruct/d2/pull/817)
```

Comment:
Could you clarify here for users that they were supported before but now are measured correctly.